src/patternfly/components/Page/page.scss

```diff
@@ -280,6 +290,36 @@
       display: block;
       visibility: visible;
     }
+
+    > .pf-c-button {
```

Comment:
These can also be dropdowns. I wonder if someone would put something else in there? Maybe we should style the header tools item in https://github.com/patternfly/patternfly-next/pull/1925 instead?

<img width="531" alt="Screen Shot 2019-07-10 at 1 31 59 PM" src="https://user-images.githubusercontent.com/35148959/60995113-7b511f00-a317-11e9-981e-644e2b6d202f.png">